testing-internal/src/main/java/com/linecorp/centraldogma/testing/internal/ProjectManagerRule.java

```diff
@@ -73,9 +74,10 @@ public CommandExecutor executor() {
     protected final void before() throws Throwable {
         super.before();
 
-        final Executor worker = newWorker();
-        projectManager = newProjectManager(worker);
-        executor = newCommandExecutor(projectManager, worker);
+        final Executor repositoryWorker = newWorker();
+        final Executor purgeWorker = Executors.newSingleThreadExecutor();
```

Comment:
- You need to stop it in `after()`.
- Please use a daemon thread.

Reply:
fix it :-)